.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+def generate_bar_chart(value, width):
+    filled = int(round(value * width / 6.0))
+    return '█' * filled + '░' * (width - filled)
+
+def generate_sparkline(data):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)